dsc/cli.py

```diff
@@ -117,4 +131,8 @@ def finalize(ctx: click.Context, email_recipients: str) -> None:
     """Process the result messages from the DSS output queue according the workflow."""
     workflow = ctx.obj["workflow"]
     workflow.process_results()
-    workflow.report_results(email_recipients.split(","))
+
+    if ctx.obj["create_and_send_report"]:
```

Comment:
I think this boolean check is better suited to `submit`, I have a hard time imagining a workflow where the app wouldn't report out the results of the DSS run. 

Also, if the `create_and_send_report` param is moved to `submit`, it could look this:

```
if create_and_send_report:
    workflow.send_report(
            report_class=FinalizeReport, email_recipients=email_recipients.split(",")
        )
```

And it could probably also be simplified to just `report`